transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -361,17 +370,74 @@ public void init(IEditorSite site,
             if (!latestVersion) prefs.setValue(VERSION_PREFERENCE, version);
 
             // Ensure Maven will compile transformations folder
-            final IProject project = manager.project();
+            IProject project = manager.project();
```

Comment:
I'll separate out this change for a test, but as an aside...

I know many people talk about having separate methods for readability, but in all my experience, the opposite is achieved by separating the logic into lots of smaller methods, which appear who knows where within the code, just for readability and not reusability.  However, it does very much lend itself to testability.